packages/react/src/theme/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { colors } from './colors'
+import * as colors from 'sfgov-design-system/src/tokens/style-dictionary/colors'
```

Comment:
we'll want to wait to merge this till we settle on our new package structure; currently, this breaks since there's no `sfgov-design-system` package anymore :)

Reply:
plus we can clean up the imports and not have such a long, winding path to get the tokens